Plugins/VendorSQLite/Process+SimpleRunPopen.swift

```diff
@@ -0,0 +1,35 @@
+#if canImport(Darwin)
+
+import Foundation
+
+extension Foundation.Process {
+    static func run(_ command: String, _ arguments: String...) throws { try Self.run(command, arguments) }
+    static func run(_ command: String, _ arguments: [String]) throws {
+        guard command.starts(with: "/") else { return try Self.run("/usr/bin/env", [command] + arguments) }
+        let process = try Self.run(URL(fileURLWithPath: command, isDirectory: false), arguments: arguments)
+        process.waitUntilExit()
+        guard process.terminationStatus == 0 else {
+            throw VendoringError("'\(command) \(arguments.joined(separator: " "))' exited with status \(process.terminationStatus)")
+        }
+    }
+    
+    static func popen(_ command: String, _ arguments: String...) async throws -> String { try await Self.popen(command, arguments) }
```

Comment:
`popen` seems to be unused?

Reply:
Incorrect; it gets used when calling `nm` and `sha3sum`.